### ISSUE TITLE
/D:/flutter/.pub-cache/hosted/pub.dartlang.org/dio-4.0.0/lib/src/dio_mixin.dart:678:8: Error: 'icthrow' isn't a type.  icthrow assureDioError(e, reqOpt);

### ISSUE THREAD

**Issue by arivu95** [flutterchina/dio#1238]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |  Web    |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |   Windows 10   |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  |   ^4.0.0   |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by arivu95** [comment]
While I try to integrate API  with the new flutter project with Dio I am getting this error from the package, why is it arising??

**Comment by suganya12121994** [comment]
I also having same error while doing Api integration with flutter

**Comment by irfan-429** [comment]
Does anyone fix this issue? I'm also having the same with `dio 4.0.0`

**Comment by irfan-429** [comment]
I just resolved the issue by deleting **dio** folder(s) from the path in my system directory where Flutter is installed `USER_NAME\Flutter\pub-cache\hosted\pub.dartlang.org` and did a flutter pub get and rebuild the project and it worked fine. 

Note: `pub-cache` folder could be hidden under Flutter folder. Show the hidden files and folders if you are unable to find.

**Comment by arivu95** [comment]
Thanks @irfan-429  will check and let u know

**Comment by arivu95** [comment]
@irfan-429  Thanks, it worked

**Comment by irfan-429** [comment]
Glad to know. You can close the issue if it really worked.
